src/hotspot/share/runtime/synchronizer.cpp

```diff
@@ -118,312 +217,13 @@ int dtrace_waited_probe(ObjectMonitor* monitor, Handle obj, Thread* thr) {
 #define NINFLATIONLOCKS 256
 static volatile intptr_t gInflationLocks[NINFLATIONLOCKS];
 
-// global list of blocks of monitors
-PaddedObjectMonitor* ObjectSynchronizer::g_block_list = NULL;
+MonitorList ObjectSynchronizer::_in_use_list;
+// Start the ceiling with one thread:
```

Comment:
This relates to me not understanding what this ceiling is (as commented elsewhere) but why does this say "start with one thread" when the value of AvgMonitorsPerThreadEstimate defaults to 1024 ??

Reply:
The estimate is that a single thread will generate at most 1024 inflated
ObjectMonitors on average.

I changed the comment like this:
```
// Start the ceiling with the estimate for one thread:
jint ObjectSynchronizer::_in_use_list_ceiling = AvgMonitorsPerThreadEstimate;
```
Does that help?